data-serving/data-service/api/openapi.yaml

```diff
@@ -40,6 +40,13 @@ paths:
           required: false
           schema:
             type: string
+          examples:
+            "full text search":
+              value: "this -butnotthis"
+              summary: Full text search with items that must or must not be present.
+            keywords:
```

Comment:
Should keywords be in quotes like "full text search" is?

Reply:
if they are multi-words yes, otherwise this is not needed. added a clarification comment.